package.json

```diff
@@ -53,6 +53,7 @@
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
     "kcd-scripts": "^0.32.1",
+    "lodash.isequal": "^4.5.0",
```

Comment:
@damonbauer - I'd prefer to keep the dependencies on this project to a minimum. Since this is only used in one spot, let's just replace it and remove the dependency.